packages/common/src/validator/index.ts

```diff
@@ -0,0 +1,24 @@
+import Joi from "joi";
+import BadRequestException from "../exceptions/BadRequestException";
+
+export default (data: object, schema: Joi.ObjectSchema, options?: Joi.ValidationOptions) => {
```

Comment:
A gente pode usar generic aqui
```suggestion
export default <T>(data: object, schema: Joi.ObjectSchema, options?: Joi.ValidationOptions): T => {
```

Dai na utilização a gente não precisaria fazer o cast